package/deb/vmagent.control

```diff
@@ -0,0 +1,10 @@
+Package: vmagent
+Maintainer: Aliaksandr Valialkin
```

Comment:
I think it would be better changing the maintainer for deb packages to @zonk1 , @kmansoft  or @patsevanton , since I'm not familiar with deb packaging.

Reply:
Changed.